### ISSUE TITLE
Composes Order

### ISSUE THREAD

**Issue by scottwarren** [brigade/scss-lint#697]
We have the declaration order as an error in our rule set, but once we add 'composes' (which as I understand it, needs to be first) the errors start rolling in. Is there anyway we can tell SCSS-Lint to ignore composes?

/cc @dane-harnett

**Comment by lencioni** [comment]
I assume you are referring to `composes` from CSS modules? You are correct that needs to be first. However, since CSS modules is relatively new, none of SCSS-Lint's rules are set up to handle it properly. We would be open to reviewing a PR that improves compatibility with CSS modules.

**Comment by scottwarren** [comment]
Yep that's what I mean, I will look into creating this as a feature, but be aware Ruby isn't my first language :+1: :smile:

**Comment by outdooricon** [comment]
We're running into this now as well, it's a really big downer. Any hints on how to add a temp ignore until there's an actual rule for it?

**Comment by outdooricon** [comment]
The extra_properties solution takes care of one issue, but it still will complain if multiple `composes` are set inside of a single selector
